ibm/service/kubernetes/data_source_ibm_container_alb.go

```diff
@@ -38,20 +38,42 @@ func DataSourceIBMContainerALB() *schema.Resource {
 				Computed:    true,
 				Description: "set to true if ALB needs to be enabled",
 			},
+			"zone": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "ALB zone",
+			},
+			"state": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "ALB state",
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Status of the ALB",
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The type of Ingress image that you want to use for your ALB deployment.",
```

Comment:
The type is either `public` or `private` so I do not think this a valid wording.
```suggestion
				Description: "the version of the ALB image",
```

Maybe a pointer could be nice to acquire the current supported images, in the description.



And I believe we should change the `alb_type` description to a more meaningful one like: `the type of ALB, available options: public, private` but since it is not part of this change I can't comment there.